pipelines/README.md

```diff
@@ -295,74 +292,17 @@ If `outputParquetPath` is set, there are additional parameters:
   [`debezium documentation`](https://camel.apache.org/components/latest/debezium-mysql-component.html)
   for more details.
 
-## Using Docker compose
-
-You can run the entire pipeline using Docker containers by following these
-instructions.
-
-### Run OpenMRS and MySQL using Docker
-
-```
-$ docker-compose -f docker/openmrs-compose.yaml up
-```
-
-Once running you can access OpenMRS at <http://localhost:8099/openmrs/> using
-username "admin" and password "Admin123". The Docker image includes the required
-FHIR2 module and demo data. Edit `docker/openmrs-compose.yaml` to change the
-default port.
-
-**Note:** If `docker-compose` fails, you may need to adjust file permissions. In
-particular if the permissions on `mysqld.cnf` is not right, the `datadir` set in
-this file will not be read by MySQL and it will cause OpenMRS to require its
-`initialsetup` (which is not needed since the MySQL image already has all the
-data and tables needed):
-
-```
-$ docker-compose -f docker/openmrs-compose.yaml down -v
-$ chmod a+r docker/mysql-build/mysqld.cnf
-$ chmod -R a+r ./utils
-$ docker-compose -f docker/openmrs-compose.yaml up
-```
-
-In order to see the demo data in OpenMRS you must rebuild the search index. In
-OpenMRS go to **Home > System Administration > Advanced Administration**. Under
-**Maintenance** go to **Search Index** then **Rebuild Search Index**.
-
-### (Optional) Run HAPI FHIR server using Docker
-
-This is only needed if you want to test sending data to a FHIR server and is not
-needed if you are only generating Parquet files.
-
-```
-$ docker-compose -f docker/sink-compose.yml up
-```
-
-You can access the FHIR server at <http://localhost:8098/fhir/>. Edit
-`sink-compose.yml` to change the default port.
-
-### Configure docker-compose.yaml
-
-Parameters (e.g port/url) have been configured to point to
-http://localhost:8099/openmrs/
-
-Remember to appropriately change other parameters such as source url, sink path,
-and any required authentication information.
-
-### Run a Transfer Pipeline
-
-#### Batch Pipeline
-
-```
-$ mvn clean install
-$ docker-compose -f docker/docker-compose.yaml up --build batch
-```
-
-##### Streaming Pipeline (Debezium)
-
-```
-$ mvn clean install
-$ docker-compose -f docker/docker-compose.yaml up --build streaming
-```
+- **What do I do if running `docker-compose` to stand up the OpenMRS image
+  fails** If `docker-compose` fails, you may need to adjust file permissions. In
+  particular if the permissions on `mysqld.cnf` is not right, the `datadir` set
+  in this file will not be read by MySQL and it will cause OpenMRS to require
+  its `initialsetup` (which is not needed since the MySQL image already has all
+  the data and tables needed):
+  `$ docker-compose -f docker/openmrs-compose.yaml down -v $ chmod a+r docker/mysql-build/mysqld.cnf $ chmod -R a+r ./utils $ docker-compose -f docker/openmrs-compose.yaml up` -**How
```

Comment:
```suggestion
  `$ docker-compose -f docker/openmrs-compose.yaml down -v $ chmod a+r docker/mysql-build/mysqld.cnf $ chmod -R a+r ./utils $ docker-compose -f docker/openmrs-compose.yaml up`
 - **How
```

Reply:
Clarification: in the previous "How" was on the previous line, after the answer to the previous question.